README.md

```diff
@@ -22,28 +22,60 @@ Our project consists of two main parts. The first part of our project is data vi
+
+## Installation
+
+To use this app, you must first download the files. To do so, open up a terminal on your personal machine, navigate to the folder in which you'd like the files to exist, and run these commands
+- `git clone https://github.com/nathanljustin/teamwork-analysis.git`
+- `cd teamworkApp`
+- `rails db:migrate`
+- `rails s`
+
+Then go to your web browser of choice, and navigate to `localhost:3000`. You are now free to use the teamwork analysis app!
 
 ## Functionality
 
-* We will provide an interface where, after submitting their survey, users can see any relevant charts, graphs, and information.
-* We will have an analysis of survey results that is useful in helping students find compatible teammates and interesting patterns or characteristics in the data that the instructor can use. The analysis will be backed up by research.
-* We will provide algorithms that can interpret the data in multiple ways.
+* Import Data: The foundation of our Teamwork Survey web app is the data, so we must have an easy way to put data in the database. When a user starts using the app, the database will be empty. The user’s only option will be to import data from a .csv file into the database, via a button on the main page. The web app assumes that the .csv file is set up such that it was an output from the [Teamwork Survey](https://docs.google.com/forms/d/1px6yEtC9zEG469OELr_1apL2iTUqwwDRYkoirncgVBI/edit) sent out by the muddersOnRails team. 
 
-## Architecture
+* Overall Distribution: Once students have been loaded into the database, a user can then examine the overall distribution of scores among them. Upon clicking “Overall distribution,” Python will generate a graph that Ruby will subsequently display for the user. The primary, secondary, tertiary, and quaternary types are displayed for each of the students in a split bar graph.
+
+* Individual Statistics: In approaching team-building, a user will want to look at which students they have and what those students’ types are. If a user has particular students in mind for a team or project, they can examine their types. On the "Individual Statistics" page, the user can select any number of students and see their scores. If more than six students are selected, an average will be displayed. Otherwise, the scores for each of the students will be displayed in a split bar graph. 
 
-The data processing will be done with Python using numpy. The processor will access the database which will be implemented in SQLite while we discuss data privacy concerns. Ruby will use the Python scripts to display the resulting graphs on a web page a user can access and add data too.
-There will be a simple database to hold JSON strings, some Python scripts that access the data to build graphs using numpy, and a Ruby page to display these graphs when asked for. Initially, there will only be one graph that Python needs to construct and Ruby needs to display.
+## Known Problems
```

Comment:
Have we figured out what was happening with the checkboxes when the same spreadsheet was imported twice? If we haven't, we add that to this section.